packages/ui/src/theme/css/styles.scss

```diff
@@ -4,21 +4,57 @@
 
 @import 'dist/theme';
 
-[data-amplify-theme] {
+// Keeping these selector low (001) so it can be easily overridden
+html {
   font-family: var(--amplify-fonts-default-static);
 }
+
 @supports (font-variation-settings: normal) {
-  [data-amplify-theme] {
+  html {
     font-family: var(--amplify-fonts-default-variable);
   }
 }
 
+// https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior
+html:focus-within {
+  scroll-behavior: smooth;
+}
+
+// https://piccalil.li/blog/a-modern-css-reset/
+body {
+  min-height: 100vh;
+  text-rendering: optimizeSpeed;
+  line-height: var(--amplify-line-heights-medium);
+}
+
 // Note: This rule can be easily overwritten when
 // needed due to very low specificity of 0 0 0
 * {
   box-sizing: border-box;
 }
 
+input,
+button,
+textarea,
+select {
+  font: inherit;
+}
+
+@media (prefers-reduced-motion: reduce) {
+  html:focus-within {
+   scroll-behavior: auto;
+  }
+  
+  *,
+  *::before,
+  *::after {
+    animation-duration: 0.01ms !important;
```

Comment:
Are we should we have to use `!important` here? If possible, we may prefer to save it as the last resource. https://css-tricks.com/when-using-important-is-the-right-choice/

This case feels like we need to use `!important`, but I'm not sure, so just to double-check 😁

Reply:
Yea I should have added a comment here, we can maybe remove those styles. I got them from a CSS reset: https://piccalil.li/blog/a-modern-css-reset/ it does force our customers to make inclusive apps (respecting reduced motion settings), but maybe it is a little heavy-handed